Containerfile

```diff
@@ -1,54 +1,38 @@
 FROM ubuntu
 
-LABEL maintainer="vchalla@redhat.com"
-
+# setup installation directory and disable prompts during installation
 WORKDIR /tmp
 ARG DEBIAN_FRONTEND=noninteractive
 
-# ALPINE
+# Install necessary libraries for subsequent commands
 RUN apt-get update
 RUN apt-get upgrade  -y
-RUN apt-get install -y podman
-RUN apt-get install -y wget python3.6 git dumb-init
-
-# Snafu Build Dependencies
-# TODO: A lot of the *-dev dependencies were added while attempting to build
-#       snafu's requirements (mainly numpy, scipy, pandas). Now that these are
-#       installed using Alpine's package manager, they may not all be necessary
-#       anymore.
-RUN apt-get install -y python3-numpy python3-scipy python3-pandas gcc python3-dev \
-                       postgresql-client libffi-dev libxml2 \
-                       libxml2-dev libxslt-dev libjpeg-dev \
-                       zlib1g-dev musl-dev
+RUN apt-get install -y podman wget git dumb-init python3.6 python3-distutils python3-pip python3-apt
 
-# RUN apt-get install -y libressl-dev
-# Install required third-party packages
+# Install vegeta for HTTP benchmarking
 RUN wget https://github.com/tsenart/vegeta/releases/download/v12.8.3/vegeta-12.8.3-linux-amd64.tar.gz
 RUN tar -xzf vegeta-12.8.3-linux-amd64.tar.gz
 RUN mv vegeta /usr/local/bin/vegeta
 
+# Install clairctl to get clair manifests
 RUN wget https://github.com/quay/clair/releases/download/v4.6.0/clairctl-linux-amd64
 RUN chmod 777 clairctl-linux-amd64
 RUN mv clairctl-linux-amd64 /usr/local/bin/clairctl
 
-
-RUN apt-get update && apt-get install -y python3.6 python3-distutils python3-pip python3-apt
-RUN apt-get update && apt-get install -y redis-server
-RUN apt-get update && apt-get install -y jq curl
-RUN ln -s /usr/bin/python3 /usr/bin/python
+# Install and setup snafu for storing vegeta results into ES
 RUN mkdir -p /opt/snafu/
 RUN git clone https://github.com/cloud-bulldozer/benchmark-wrapper
 RUN cd benchmark-wrapper && \
     cp -R . /opt/snafu/
 RUN pip3 install --upgrade pip
 RUN pip3 install -e /opt/snafu/
 
-# Install Python Dependencies
-# RUN python3 -m ensurepip
-RUN python3 -m pip install --upgrade pip
+# Cleanup the installation remainings
 RUN apt-get clean autoclean
 RUN apt-get autoremove --yes
 RUN rm -rf /var/lib/{apt,dpkg,cache,log}/
```

Comment:
this `rm` isn't helpful

Reply:
The above 3 lines are useful for the cleanup and they will reduce some amount of container size.

* The first command `RUN apt-get clean` autoclean removes packages that were automatically installed to satisfy dependencies and are no longer needed. This frees up disk space in the container.
* The second command `RUN apt-get autoremove --yes` removes packages that were installed as dependencies for other packages, but are no longer needed by any package. This also frees up disk space.
* The third command `RUN rm -rf /var/lib/{apt,dpkg,cache,log}/` removes cached package files, temporary files, and logs from the `/var/lib/apt, /var/lib/dpkg, and /var/log` directories. This helps reduce the size of the container and keeps it clean.

Please feel free to share your concerns on this.

Reply:
we usually attempt to reduce the image size for most of our container builds -- if there is a better way to do this, let us know!

Reply:
You need to remove files in the same step they're created, otherwise they just make the container larger.

Reply:
Acknowledged. Moved the cleanup command along with `apt-get update` and `apt-get install` as suggested. Thanks!